src/index.ts

```diff
@@ -138,26 +138,26 @@ client.on('ready', async () => {
             log(type, `${chalk.yellow('[')}${chalk.yellow.bold('MODLOAD')}${chalk.yellow(']')} ${message}`);
```

Comment:
```suggestion
            log(type, `${chalk.yellow('[')}${chalk.yellow.bold('HOOKLOAD'}${chalk.yellow(']')} ${message}`);
```

`MODLOAD` is "Module Load", not "Hook Load".